README.md

```diff
@@ -37,3 +37,13 @@ MONITOR_TOKEN=[YOUR API TOKEN IN JENKINS]
 * Run the app by typing `rackup`
 
 It will fireup a web server and can be accessed at [localhost:4567](http://localhost:4567)
+
+## Docker  
+
+### Running  
+
+Running Jenkins Pipeline on Docker is simple as `docker run -p 4567:4567 -e MONITOR_USERNAME=<your-username> -e MONITOR_TOKEN=<your-token> -v <your-path-to-config>/jenkins-pipeline/config jenkinspipeline/jenkins-pipeline`
```

Comment:
There is a way we can me make it simpler to run the command? may as part of the package.json?